src/config.js

```diff
@@ -43,6 +43,10 @@ global.config = {
     //screepsplusToken: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InRvb2FuZ2VscyIsImlhdCI6MTQ4MzU2MTU3OSwiYXVkIjoic2NyZWVwc3BsLnVzIiwiaXNzIjoic2NyZWVwc3BsLnVzIn0.NhobT7Jg8bOAg-MYqrYsgeMgXEVXGVYG9s3G9Qpfm-o'
   },
 
+  debug: {
+    getPartsConfLogs: false
```

Comment:
I like this, we should introduce some kind of logging, so that we can easily enable and disable certain feature logs.

Not for this PR, just an idea